Makefile

```diff
@@ -52,50 +61,51 @@ CLINAME          := $(PRODUCT_NAME)
 
 # Release versions of the CLI are built in three phases:
 #
-#    1) TMP_BUILD             - No build metadata.
+#    1) INITIAL_BUILD       - No build metadata.
 #    2) INTERMEDIATE_BUILD    - Some build metadata.
-#    3) RELEASE_BUILD         - All build metadata.
+#    3) BOOTSTRAPPED_BUILD           - All build metadata.
```

Comment:
I renamed all these for hopefully the last time... I think we're consistent with naming everywhere now.